### PR TITLE
Download tracks that have been commented by user 

### PR DESCRIPTION
**Pull request by devalnor**
Resolve issue: https://github.com/flyingrub/scdl/pull/213

Remarques:

- The API for https://api.soundcloud.com/users/{id}/comments do not provide other information than the track_id so I create `get_track_info` who is not the same feature than `resolver`. I did in your main `scdl.py` file but I guess that `client.py` may be a better place in the future.

- Probably capital option '-C' is not the best option and I suggest a refactor of the options or just a switch with continue -C is more logical for it.

- Tested this time ;)

### CODE REVIEW

File: scdl/scdl.py [hunk 1]
```diff
@@ -298,6 +315,9 @@ def download(user, dl_type, name):
                 download_playlist(item)
             elif dl_type == 'playlists-liked':
                 parse_url(item['playlist']['uri'])
+            elif dl_type == 'commented':
+                item=get_track_info(item['track_id'])
+                download_track(item)
```
Comment (flyingrub) [inline scdl/scdl.py]: You are downloading only one track here ?

Comment (devalnor) [inline scdl/scdl.py]: Nope.
Ok you I understand, you want to use parse_url for track list?

Comment (flyingrub) [inline scdl/scdl.py]: item here is actually a collection. You cannot directly pass it to download_track.
Did you tried to run this code ?

Comment (devalnor) [inline scdl/scdl.py]: Of course I did...

Every commented tracks are a in the `ressources` collection. And like I said in my comment, the API for `https://api.soundcloud.com/users/{id}/comments` do not provide other information than the `track_id` in the item collection. 

So to resolve it, I overwrite the item with a new `get_track_info(track_id)` and call download_track like in your else condition.

Comment (flyingrub) [inline scdl/scdl.py]: Wooops. My bad, I totaly missread your PR :S

**Comment by flyingrub** [comment]
Thanks for your quick PR for #215

**Comment by winterflags** [comment]
@flyingrub @devalnor 

Trying this but doesn't work, it just throws the usage instructions: 

`scdl -l https://soundcloud.com/hurfyd -C -c`

Using latest version of scdl from pip:

```
scdl --version
v1.6.6
```

Am I missing something here or maybe incorrect syntax?
I am using a personal AUTH_TOKEN for scdl if that has anything to do with it.

**Comment by flyingrub** [comment]
The version was not updated on pipy. It is now available on v1.6.7.

**Pull request merged**